src/main/java/utils/RoundParser.java

```diff
@@ -0,0 +1,25 @@
+package utils;
+
+public class RoundParser {
+
+    public static final String INVALID_ROUND_INPUT = "[ERROR] 시도 횟수는 1 이상의 숫자여야 합니다.";
+
+    public static int parse(String input) {
+        int round;
+
+        try {
+            round = Integer.parseInt(input.trim());
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(INVALID_ROUND_INPUT);
+        }
+
+        validate(round);
+        return round;
```

Comment:
```suggestion
        try {
            int round = Integer.parseInt(input.trim());
            validate(round);
            return round;
        } catch (NumberFormatException e) {
            throw new IllegalArgumentException(INVALID_ROUND_INPUT);
        }
```

굳이 재할당을 하지 않고도 만들 수 있을 것 같아요!

Reply:
피드백 감사합니다~! 🙇‍♂️  

말씀해주신 것처럼 `round` 변수를 먼저 선언하고 나중에 할당하기보다는,  
try 블록 안에서 바로 선언과 초기화를 함께 해주는 방식이 더 깔끔하고 불필요한 상태를 줄일 수 있다는 점에 공감이 됩니다!

예전에는 try 블록 안에서 변수를 선언하는 게 어색하게 느껴져서 바깥에 미리 선언해두는 습관이 있었던 것 같은데,  
이번 피드백을 계기로 재할당 없이 바로 처리하는 방식이 더 명확하고 안정적인 흐름이라는 걸 다시 느끼게 되는 것 같습니다.

다음부턴 이런 부분도 더 신경 써서 코드를 정리해보겠습니다~!  
좋은 포인트 짚어주셔서 감사합니다 😊